src/platform/plugins/shared/metrics_experience/server/routes/dimensions/get_dimensions.test.ts

```diff
@@ -0,0 +1,307 @@
+/*
```

Comment:
We already have a functional test covering this, btw.

Reply:
yes, the one failing 😅 
I can remove delete this unit test then